packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/drawNode.css

```diff
@@ -74,6 +74,16 @@ limitations under the License.
   padding: 0.3rem 0.5rem 0.3rem 0.75rem;
 }
 
+.DiffNode--popover .DiffNode--copyIcon,
+.DiffNode:not(:hover) .DiffNode--copyIcon {
+  display: none;
+}
+
+.DiffNode--labelCell {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
Can you merge this with the CSS class on [LOC 73](https://github.com/jaegertracing/jaeger-ui/pull/312/files#diff-d6dac11b88f602aa428194a0a0179eb3R73)?